ClientApp/src/Pages/WhaleSpecies/WhaleSpecies.scss

```diff
@@ -1,2 +1,216 @@
 @import "../../Styles/constants";
-@import "../../Styles/styles";
\ No newline at end of file
+@import "../../Styles/styles";
+
+@mixin aspect-ratio($width, $height) {
+  position: relative;
+
+  &:before {
+    display: block;
+    content: "";
+    width: 100%;
+    padding-top: ($height / $width) * 100%;
+  }
+
+  > img {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    width: 100%;
+    height: 100%;
+  }
+}
+
+// Styling
+
+section {
```

Comment:
could we change any top level css selectors in this file that target generic html elements to target classes instead please? Contrary to what sounds sensible the styling in this file will actually get applied to every page, so any generic html element selector in this file that isn't inside something like .carousel could be picked up by a similar html element on another page :)